cmd/install/install.go

```diff
@@ -736,3 +768,23 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, error) {
 
 	return objects, nil
 }
+
+func getImageRegistryCABundle() string {
+	if client, err := util.GetClient(); err == nil {
```

Comment:
My preference would be to gate this on a capability (check whether the image.config.openshift.io resource exists in the mgmt cluster) and report error if there's an error, instead of silently ignoring any errors that occur.

Reply:
Thank you for the review! I've added a capability check [here](https://github.com/openshift/hypershift/pull/2713/files#diff-b105e0fd290117fe4056a1e2270006e9ea7a6025f5b57936dff42d50660e0b9cR492) and have logged the missing capability [here](https://github.com/openshift/hypershift/pull/2713/files#diff-b105e0fd290117fe4056a1e2270006e9ea7a6025f5b57936dff42d50660e0b9cR524). 

How does it look?